interop.h

```diff
@@ -0,0 +1,15 @@
+#ifndef V8GO_INTEROP_H
+#define V8GO_INTEROP_H
+
+#include "errors.h"
+#include "forward-declarations.h"
+
+// A go-friendly return type combining value and error.
+// Maybe this really belongs in value.h, but seems to be used from multiple
+// places.
+typedef struct {
+  ValuePtr value;
+  RtnError error;
+} RtnValue;
```

Comment:
Consider moving this to errors.h. It only exists to add an error to a value, so I think it belongs in the realm of error handling (and this file is small, with a vague scope.)

Reply:
Yeah, I had trouble placing this. I didn't think it so much of as an error, but more like converting the result to idiomatic go.

But I wouldn't mind putting it in errors. I didn't look too much at what the "errors.h" really contains, as extracting the errors file was mostly the result of, if I move A, I also need to move B.

Reply:
My problem is this file is too small and unfocused: no one will think of looking at this file to find RtnValue. A file that is about mapping error handling domains makes more sense to me. That said, it's not a strong opinion.

Reply:
I've moved it to errors.h for now. Maybe a sensible place will reveal itself later :) 

I don't have a strong opinion either.

Reply:
Btw, on "small files", there is also `utils.h`/`utils.cc` containing string copying functionality ...

Opinion on those?

Reply:
I think they're fine. Having one slush bucket is better than two, and probably unavoidable. :)

Reply:
Should we remove the almost empty file?

Reply:
Damn, I thought I did that.